dune

```diff
@@ -296,23 +296,48 @@
    %{targets}
    %{dep:middle_end/flambda2/ui/flambda2_ui.a}
    %{dep:middle_end/flambda2/algorithms/flambda2_algorithms.a}
-   %{dep:middle_end/flambda2/lattices/flambda2_lattices.a}
    %{dep:middle_end/flambda2/numbers/flambda2_numbers.a}
-   %{dep:middle_end/flambda2/identifiers/flambda2_identifiers.a}
    %{dep:middle_end/flambda2/kinds/flambda2_kinds.a}
+   %{dep:middle_end/flambda2/identifiers/flambda2_identifiers.a}
    %{dep:middle_end/flambda2/nominal/flambda2_nominal.a}
+   %{dep:middle_end/flambda2/lattices/flambda2_lattices.a}
    %{dep:middle_end/flambda2/bound_identifiers/flambda2_bound_identifiers.a}
    %{dep:middle_end/flambda2/term_basics/flambda2_term_basics.a}
    %{dep:middle_end/flambda2/terms/flambda2_terms.a}
-   %{dep:middle_end/flambda2/from_lambda/flambda2_from_lambda.a}
-   %{dep:middle_end/flambda2/cmx/flambda2_cmx.a}
    %{dep:middle_end/flambda2/types/flambda2_types.a}
+   %{dep:middle_end/flambda2/cmx/flambda2_cmx.a}
+   %{dep:middle_end/flambda2/from_lambda/flambda2_from_lambda.a}
    %{dep:middle_end/flambda2/simplify/flambda2_simplify.a}
-   %{dep:middle_end/flambda2/to_cmm/flambda2_to_cmm.a}
    %{dep:middle_end/flambda2/parser/flambda2_parser.a}
    %{dep:ocamloptcomp.a}
+   %{dep:middle_end/flambda2/to_cmm/flambda2_to_cmm.a}
    %{dep:middle_end/flambda2/flambda2.a})))
 
+(rule
```

Comment:
It looks like the three rules (for `.a`, `.cma` and `.cmxa` files)
should be kept synchronized; I am not sure whether it is obvious
or deserves a comment.

Reply:
Comment added